docs/getting-started.md

```diff
@@ -47,135 +51,164 @@ Radicle development is public by default and completely open-source. See
 ```
-$ which git-remote-rad
-<home path>/.radicle/bin/git-remote-rad
+$ rad push
+Pushing 🌱 to remote `rad`
+Everything up-to-date
+
+Git version 2.35.1
+Select a seed node to sync with...
+* pine.radicle.garden
+* willow.radicle.garden
+* maple.radicle.garden
 ```
 
-With the remote helper installed, you will be able to [push changes][pc] from
-your working copies to your Radicle [monorepo][ri] or [fetch changes][fc] from
-other peers via the `rad` remote. The Upstream client then publishes
-these changes to the network.
-
-To start using Radicle, continue on to the [Using Radicle][ur] section.
-
-[ur]: using-radicle/overview.md
-[di]: understanding-radicle/glossary.md/#device-id
-[pc]: using-radicle/pushing-changes
-[fc]: using-radicle/viewing-and-merging-patches
-[rg]: understanding-radicle/faq.md#
-[ri]: understanding-radicle/how-it-works.md/#git-implementation
-[sn]: using-radicle/running-a-seed-node.md
-[cc]: get-involved/community.md
-
-[id]: /img/identity.png
-
-[as]: https://support.apple.com/en-us/HT211814
-[bs]: https://github.com/radicle-dev/radicle-upstream/blob/main/docs/development.md
-[co]: https://radicle.community
-[do]: https://radicle.xyz/downloads.html
-[gd]: https://git-scm.com/book/en/v2/Getting-Started-First-Time-Git-Setup
-[sd]: https://github.com/radicle-dev/radicle-bins/tree/master/seed
-[te]: https://radicle.xyz/terms.html
-[mc]: https://matrix.radicle.community
-[sc]: https://github.com/radicle-dev/radicle-upstream
+Once your project finishes syncing for the first time, you'll find important information about where your project lives
+on the Radicle Network.
+
+```
+Git signing key ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIL460KIEccS4881p7PPpiiQBsxF+H5tgC6De6crw9rbU
+
+Syncing 🌱 project rad:git:hnrkqi6ohci9m59i54ppiy3fqkedkjt98ymdo to https://willow.radicle.garden
+
+ok Syncing delegate identity hnrkqdpm9ub19oc8dccx44echy76hzfsezyio...
+ok Syncing project identity...
+ok Syncing project refs...
+ok Fetching remotes (*)...
+ok Project synced.
+
+🌱 Your project is synced and available at:
+
+    (web) https://app.radicle.network/seeds/willow.radicle.garden/rad:git:hnrkqi6ohci9m59i54ppiy3fqkedkjt98ymdo/
+    (git) https://willow.radicle.garden/hnrkqi6ohci9m59i54ppiy3fqkedkjt98ymdo.git
+```
+
+Note the `(web)` and `(git)` URLs, which you'll use in the next step.
+
+## View your project and share with others
+
+After you've pushed your project to the Radicle network, it's available for viewing on the Radicle [web
+client](https://app.radicle.network).
+
+To view your project, paste the URL after `(web)` into your browser of choice. You can use this interface to browse your
+code, view individual files, explore your commit history, and more. You can share this URL with others, which allows
+them to view your code on the web client, clone your code into a new project.
+
+![The Radicle web client for viewing a published
+project](https://user-images.githubusercontent.com/1153921/157982795-0bfcaa58-ec9d-44f2-8da3-ad0900172e0e.png)
```

Comment:
We should add this image to this repo and just use a relative link here.

Reply:
Fixed! Old habits die hard.